projects/uitdatabank/docs/entry-api/project-flow.md

```diff
@@ -0,0 +1,60 @@
+---
+stoplight-id: 73ef6e15ee3ec
+---
+
+# Project flow from test to production
+
+For the integration with the UiTdatabank Entry API, there is a flow in which you first obtain access to our test environment. 
+Access to production environment is given after your integration on test environment has been successful validated by publiq vzw.
+
+On this page we describe how this flow works and how you can ensure that you gain access to the production environment as quickly as possible!
+
+> Make sure you read this guide before you start developing on our test environment!
+
+## Step 1: start on test environment
+
+You can apply for free test credentials at [Projectaanvraag](https://projectaanvraag.uitdatabank.be). 
+
+Once you've obtained your personal test credentials you can start with the development of your integration with the UiTdatabank Entry API on our test environment.
+
+## Step 2: validation by publiq vzw
+
+As soon as you have finished developing your integration with the Entry API, you send a minimum of 5 of events to our test environment. You contact us via `vragen@publiq.be` and provide us the identifiers of the created test events. We will then validate your integration as soon as possible. 
```

Comment:
Should we suggest that they use a specific subject (or mention something like "content check" in the subject)?

Reply:
Good suggestion!

Reply:
I think there's now a word (or multiple) missing in:

> You contact us via `vragen@publiq.be` the email subject "content check".

Maybe something like this?

> You contact us via `vragen@publiq.be` with the email subject "content check".